flaky-tests/frameworks/mocha.md

```diff
@@ -71,10 +66,20 @@ var mocha = new Mocha({
 });
 ```
 
-## Further Configuration
 
-MochaJS is only one of several popular testing frameworks for Javascript & Typescript. You might also want to look at
+
+
+## Further Information
+See an example of MochaJS invoked form a GitHub action [here](https://github.com/trunk-io/flake-factory/blob/main/.github/workflows/javascript-tests.yaml#L29).
+
+
+
+MochaJS is only one of several popular testing frameworks for Javascript & Typescript. You might also want to look at:
 
 * [Jest](https://trunk.io/testing/jest)
```

Comment:
Should this still be here? The links seem to all point outside docs. Leftover?

Reply:
removed